index.js

```diff
@@ -41,6 +41,6 @@ function fastifyHemera(fastify, opts, next) {
 }
 
 module.exports = fp(fastifyHemera, {
-  fastify: '^2.0.0',
+  fastify: '>=2.0.0',
```

Comment:
I'd set it to >=3

Reply:
That'd break 2.x installs (which is still LTS) - So if you plan to... I'd suggest a major version bumb. Leaving that up to you. But fastify-hemera as it is, supports both.

Reply:
For v2 you can still use the previous version. I will do a major bump.

Reply:
agreed... here you go